Question5.java

```diff
@@ -0,0 +1,152 @@
+//Stack Interface
+interface Stack<Type>
+{
+    void push(Type data) throws StackSizeExceeded;
+    void pop();
+    void peek();
+    int size();
+}
+
+//Exception Classes
+class StackSizeExceeded extends Exception
+{
+    public StackSizeExceeded(String message) {super(message);}
+}
+class IllegalArgumentException extends Exception
+{
+    public IllegalArgumentException(String message) {super(message);}
+}
+
+//Limited Stack Class
+class LimitedStack<Type> implements Stack<Type>
+{
+    Node head;
+    int max_stack;
+
+    public LimitedStack(int max) throws IllegalArgumentException //LimitedStack Constructor
+    {
+        if(max <= 0) //Validate Argument
+        {
+            throw new IllegalArgumentException("Illegal Argument");
```

Comment:
Change the message to Size cannot be {given size}

Reply:
This issue has been resolved.